metadata_backend/helpers/schemas/submission.json

```diff
@@ -148,67 +148,114 @@
+                                    "3112 - Neurosciences",
+                                    "3121 - Internal medicine",
+                                    "3122 - Cancers",
+                                    "3123 - Gynaecology and paediatrics",
+                                    "3124 - Neurology and psychiatry",
+                                    "3125 - Otorhinolaryngology, ophthalmology",
+                                    "3126 - Surgery, anesthesiology, intensive care, radiology",
+                                    "313 - Dentistry",
+                                    "3141 - Health care science",
+                                    "3142 - Public health care science, environmental and occupational health",
+                                    "315 - Sport and fitness sciences",
+                                    "316 - Nursing",
+                                    "317 - Pharmacy",
+                                    "318 - Medical biotechnology",
+                                    "319 - Forensic science and other medical sciences",
+                                    "4 - Agriculture and forestry",
+                                    "4111 - Agronomy",
+                                    "4112 - Forestry",
+                                    "412 - Animal science, dairy science",
+                                    "413 - Veterinary science",
+                                    "414 - Agricultural biotechnology",
+                                    "415 - Other agricultural sciences",
+                                    "5 - Social sciences",
+                                    "511 - Economics",
+                                    "512 - Business and management",
+                                    "513 - Law",
+                                    "5141 - Sociology",
+                                    "5142 - Social policy",
+                                    "515 - Psychology",
+                                    "516 - Educational sciences",
+                                    "517 - Political science",
+                                    "518 - Media and communications",
+                                    "519 - Social and economic geography",
+                                    "520 - Other social sciences",
+                                    "6 - Humanities",
+                                    "611 - Philosophy",
+                                    "6121 - Languages",
+                                    "6122 - Literature studies",
+                                    "6131 - Theatre, dance, music, other performing arts",
+                                    "6132 - Visual arts and design",
+                                    "614 - Theology",
+                                    "615 - History and archaeology",
+                                    "616 - Other humanities",
+                                    "9 - Other",
+                                    "999 - Other"
```

Comment:
If the frontend utilizes this list of values for creating submissions, then this shouldn't break anything

Reply:
it was hardcoded as FOS https://github.com/CSCfi/metadata-submitter-frontend/blob/master/src/features/wizardSubmissionFolderSlice.tsx#L227-L228

Reply:
we can fix that in a later iteration of front-end, and probably we can alleviate that by making some changes in the backend

Reply:
it is good at is, we can address changes when front-end starts to work on it